tests/tap_tester/base.py

```diff
@@ -110,16 +117,25 @@ def get_start_date(self):
 
     def get_credentials(self):
         """Authentication information for the test account. Api secret is expected as a property."""
-        """Authentication information for the test account"""
+        
         credentials_dict = {}
-        creds = {"api_secret": "TAP_MIXPANEL_API_SECRET"}
+        if self.eu_residency_server:
+            creds = {"api_secret": "TAP_MIXPANEL_EU_RESIDENCY_API_SECRET"}
+        else:
+            creds = {"api_secret": "TAP_MIXPANEL_API_SECRET"}
+            
         for cred in creds:
             credentials_dict[cred] = os.getenv(creds[cred])
 
         return credentials_dict
 
     def expected_streams(self):
         """A set of expected stream names"""
+        
+        #Skip `export` and `revenue` stream for EU recidency server
```

Comment:
@prijendev Add reason for skipping these streams

Reply:
done